docs/application/dotnet/guides/webview/webview.md

```diff
@@ -1,5 +1,8 @@
 # WebView
 
+> [!NOTE]
+> WebView class is Deprecated since API Level 10, and will be removed in API Level 12. The WebView provided by .NET MAUI or Tizen.NUI can be used instead.
```

Comment:
```suggestion
> WebView class is deprecated since API Level 10 and will be removed in API Level 12. The WebView provided by .NET MAUI or Tizen.NUI can be used instead.
```